src/librpmem/rpmem_fip.c

```diff
@@ -1431,7 +1432,13 @@ rpmem_fip_flush_gpspm(struct rpmem_fip *fip, size_t offset,
 	 * Since for now GPSPM mode is considered auxiliary and its performance
 	 * is not critical, the flush is emulated with a persist.
 	 */
-	ret = rpmem_fip_persist_saw(fip, offset, len, lane, flags);
+	if (mode == RPMEM_PERSIST_SEND) {
```

Comment:
now rpmem_fip_flush_gpspm and rpmem_fip_persist_gpspm look exactly the same - please share this code